Loop/Managers/DeviceDataManager.swift

```diff
@@ -26,20 +27,27 @@ enum State<T> {
 class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate {
     /// Notification posted by the instance when new glucose data was processed
     static let GlucoseUpdatedNotification = "com.loudnate.Naterade.notification.GlucoseUpdated"
-
+    
```

Comment:
Do you feel strongly about the whitespace on blank lines? If so, then I'll change my settings.

Here's my current Xcode setting:
<img width="354" alt="screen shot 2016-06-24 at 6 45 14 pm" src="https://cloud.githubusercontent.com/assets/212876/16354046/d3726e52-3a3b-11e6-97f3-cc30aa1d8cab.png">